v21.2/sql-feature-support.md

```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
  Roles | ✓ | Standard | [Roles documentation](authorization.html#roles)
  Object ownership | ✓ | Common Extension | [Ownership documentation](authorization.html#object-ownership)
  Privileges | ✓ | Standard | [Privileges documentation](authorization.html#assign-privileges)
+ Default privileges | Partial | PostgreSQL Extension | [Default privileges documentation](authorization.html#default-privileges)
 
 ### Miscellaneous
 
  Component | Supported | Type | Details
 -----------|-----------|------|---------
  Column families | ✓ | CockroachDB Extension | [Column Families documentation](column-families.html)
  Computed columns (stored and virtual) | ✓ | Common Extension | [Computed Columns documentation](computed-columns.html)
+ `ON UPDATE` expressions | ✓ | MySQL Extension | [`ON UPDATE` expressions documentation](create-table.html#on-update-expressions)
  Multi-region capabilities | ✓ | CockroachDB Extension | [Multi-region documentation](multiregion-overview.html)
```

Comment:
It's a bit weird that we use title case for links to titles, and sentence case for links to headings, but I guess that's our convention.